h/static/scripts/test/bridge-test.coffee

```diff
@@ -37,7 +37,9 @@ describe 'Bridge', ->
 
     it 'adds the channel to the .links property', ->
       channel = createChannel()
-      assert.include(bridge.links, {channel: channel, window: fakeWindow})
+      assert.isTrue(bridge.links.some((link) ->
```

Comment:
See the commit message for an explanation. The reason for this change is to avoid a deep equality test on the _values_ of the `channel` and `fakeWindow` objects.